acceptance-tests.md

```diff
@@ -78,9 +93,11 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 
 1. **View and Manage Role bindings**
     * _As an administrator of the cluster, I should be able to create, view and manage role bindings for the users in the cluster._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. After a user is added, an admin can go to the user ‘actions’ tab and set their role to ‘manager’ or ‘user’.
             1. [https://nerc-project.github.io/nerc-docs/get-started/get-an-allocation/#adding-and-removing-user-from-the-project](https://nerc-project.github.io/nerc-docs/get-started/get-an-allocation/#adding-and-removing-user-from-the-project)
+    * Acceptance tests:
```

Comment:
Same question about delays for tthis to get from ColdFront to OpenShift and how to test the use case.